docs/docs/dynamo-store/access-strategies.md

```diff
@@ -4,7 +4,6 @@ The access strategy allows you to opt into custom loading or reloading
 algorithms. For most systems, `Unoptimized` is a safe default; you can always
 switch later.
 
-
 # Unoptimized
 
 The first, and likely most used access strategy is the unoptimized one. It'll
```

Comment:
The first, and likely most used access strategy is the unoptimized one, which just scans to get the events across all the items in a Partition.

Reply:
>Loading uses a `Query` that loads batches backwards from the tip
Strange how this leaked into the docs; that only comes into play with isOrigin.